src/main/java/com/cdancy/bitbucket/rest/domain/pullrequest/Reference.java

```diff
@@ -31,11 +31,14 @@
 
     public abstract MinimalRepository repository();
 
+    @Nullable
+    public abstract String displayId();
+
     Reference() {
     }
 
-    @SerializedNames({ "id", "repository" })
-    public static Reference create(String id, MinimalRepository repository) {
-        return new AutoValue_Reference(id != null ? id : "refs/heads/master", repository);
+    @SerializedNames({"id", "repository", "displayId"})
+    public static Reference create(String id, MinimalRepository repository, String displayId) {
```

Comment:
Constructor was the wrong word, my apologies. Overloaded versions of `create`:

```
public static Reference create(String id, MinimalRepository repository) {

public static Reference create(String id, MinimalRepository repository, String displayId) {
  
```

The first can delegate to the second passing in whatever value `id` is to `displayId`.